transmogrifier/sources/transformer.py

```diff
@@ -224,10 +224,45 @@ def parse_source_file(cls, source_file: str) -> Iterator[dict[str, JSON] | Tag]:
         """
         pass
 
+    @final
+    def _transform(
+        self, source_record: dict[str, JSON] | Tag
+    ) -> Optional[TimdexRecord]:
+        """
+        Private method called by Transform a source record into a TIMDEX record.
```

Comment:
This docstring reads a little awkwardly to me.

I might suggest this docstring explain this privated method is called for both XML and JSON transformations, where all logic is shared except `source_record` type?

Reply:
This was clunky and I had a word missing, fixed!